packages/2.questions-search/src/app/views/questions/question-list/question-list.component.scss

```diff
@@ -0,0 +1,45 @@
+table {
+  width: 90%;
+  margin: 1%;
+}
+
+.mat-column-delete {
+  text-align: right;
+}
+
+.create-button{
+  margin:1%;
+}
+
+@-webkit-keyframes hover_animation {
+  to { background-color: #e0e0e0; }
+}
+@-moz-keyframes hover_animation {
+  to { background-color: #e0e0e0; }
+}
+
+@-ms-keyframes hover_animation {
+  to { background-color: #e0e0e0; }
+}
+@-o-keyframes hover_animation {
+  to { background-color: #e0e0e0; }
+}
+@keyframes hover_animation {
+  to { background-color: #e0e0e0; }
+}
+
+.mat-row:hover {
+  cursor: pointer;
+  -webkit-animation: hover_animation 0.25s;
+	-moz-animation: hover_animation 0.25s;
+	-ms-animation: hover_animation 0.25s;
+	-o-animation: hover_animation 0.25s;
```

Comment:
This should be added automatically by the pre-processor. Maybe you don't need it. 😅